cleanup/action.yml

```diff
@@ -7,5 +7,6 @@ runs:
     - run: |
         set -e
         shopt -s dotglob
-        rm -rf ${{ github.workspace }}/*
+        whoami
+        rm -rfv ${{ github.workspace }}/*
```

Comment:
Honestly, this looks useless. The `-v` flag shoes which files were removed, but is not interesting for us, more interesting which files couldn't be removed.

```
rm -rf ${{ github.workspace }}/* || { echo "Files: $(ls -Rla)"; exit 1; }
```

In such a way we could see which files were left not removed. Also, it would be nice to add a comment with the link to the bug and some explanation that it is for debug purposes.